src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -68,10 +68,10 @@ def validate_location(cmd, location):
             break
 
 
-def get_chart_path(registry_path, kube_config, kube_context):
+def get_chart_path(registry_path, kube_config, kube_context, helm_client_location):
     # Pulling helm chart from registry
     os.environ['HELM_EXPERIMENTAL_OCI'] = '1'
-    pull_helm_chart(registry_path, kube_config, kube_context)
+    pull_helm_chart(registry_path, kube_config, kube_context, helm_client_location)
```

Comment:
Do we need to consider setting a default value as `helm` for parameter `helm_client_location`?

Reply:
helm_client_location will always be set to a value (local path in .azure directory) before passing to other functions.
We execute the function 'helm_client_location = install_helm_client()' in every cmd grp before passing 'helm_client_location' to other functions

Reply:
We do not want the default to be "helm" because we do not want to use the helm binary installed on user's machine. But install a helm version compatible with our CLI. So we want to avoid that

Reply:
Got it, thanks~